Duckling/Quantity/EN/Corpus.hs

```diff
@@ -111,4 +129,12 @@ allExamples = concat
               , "over four ounces of chocolate"
               , "more than four ounces of chocolate"
               ]
+  , examples (simple Tonne 2 Nothing)
+              [ "2 tonnes"
+              , "2 tons"
+              ]
+  , examples (simple Tonne 3 (Just "sugar"))
+              [ "3 tonnes of sugar"
+              , "3 tons of Sugar"
+              ]
```

Comment:
Maybe a test case for `1 tonne` or `a tonne` and also the abbreviation `t`

Reply:
The procedure that infers the number from `a` or `an` is different and I have not touched it in my changeset. I guess it works regardless of what is being put in front of it. Maybe the test for it is elsewhere. I will look for its test and update the issue.